python/xorbits/_mars/_utils.pyx

```diff
@@ -142,6 +142,9 @@ cdef inline list iterative_tokenize(object ob):
 cdef inline tuple tokenize_numpy(ob):
     cdef int offset
 
+    if ob.nbytes > 64 * 1024 ** 2:
+        return os.urandom(16), ob.dtype, ob.shape, ob.strides
```

Comment:
Could you add some comments to explain this behavior?

Reply:
Deal with the segmentation fault that occurs when the numpy size is too large.
Do I need to put comments in the code as well?

Reply:
I know, I mean you can add some code comments to explain the reason, it's helpful when others want to understand these codes.

Reply:
done